shimmingtoolbox/cli/download_data.py

```diff
@@ -3,19 +3,27 @@
 import os
 import logging
 import click
+from typing import Dict, Tuple, List
 
 from shimmingtoolbox.download import install_data
 
-dict_url = {
-    "testing_data": ["https://github.com/shimming-toolbox/data-testing/archive/r20200713.zip"],
-    "prelude": ["https://github.com/shimming-toolbox/binaries/raw/master/prelude"]
+# URL dictionary is in the format:
+# - key: name of item to download
+# - value: tuple containing:
+#     0. List containing the item's URL string
+#     1. String description of the item
+
+URL_DICT: Dict[str, Tuple[List[str], str]] = {
+    "testing_data": (["https://github.com/shimming-toolbox/data-testing/archive/r20200713.zip"], "Light-weighted dataset for testing purpose."),
```

Comment:
Split in two line for PEP8 compatibility.

Reply:
Resolved in a9f042a